docs/monoDrive_home/Weather.md

```diff
@@ -71,139 +54,167 @@ window for the Actor. Scroll down to the "Weather Description" in the "Details"
+**ZenithColor**: Color of the upper portion of the sky.  
+**HorizonColor**: Color of the lower portion of the sky.  
+**OverallSkyColor**: Overall color of the sky blended with horizon and zenith.  
+**SkyLightIntensity**: Ambient lighting applied to the scene. This is used in addition to the directional light to add ambient lighting.  
+**SkyLightColor**: The color of the ambient light.  
+
+### Fog  
+**FogDensity**: How dense the fog is from 0.0 to 0.05  
+**SecondFogDensity**: A second layer fog density, improves depth, from 0.0 to 0.05  
+**FogHeightFalloff**: Height at which the fog stops, from 0.0 to 2.0  
+**SecondFogHeightFalloff**: Height at which the second fog stops, from 0.0 to 2.0  
+**SecondFogHeightOffset**: Increasing drastically drops off the visibility. 0.0 to inf  
+**FogInscatteringColor**: Fog color. RGBA (0.0->1.0)  
+**FogMaxOpacity**: How clear the fog is. 0.0 to 1.0  
+**FogStartDistance**: Distance from viewer that the fog starts. 0.0 to 5000.0  
+**FogCutoffDistance**: Distance from the viewer that the fog stops. 0.0 to 20000000.0  
+
+### Precipitation Effects
+**bPrecipitation**: Weather to include rain or wetness on master road material.  
+**PrecipitationType**: Type of precicipiation; only Rain is currently supported.  
+**PrecipitationAmount**: Amount of precipitation to display in the the particle effect. Also effects sensors like lidar which will have more attenuation. 0.0 to 100.0  
+**PrecipitationAccumulation**: Amount of precipitation to accumulate on the ground. Visibile in amount of water on ground. 0.0 to 100.0  
+
+### Wind Effects  
+**bWind**: Whether to include wind affects such as swaying of trees and foliage.  
+**WindIntensity**: How intense the wind affect is from 0.0 to 100.0.  
+**WindAngle**: Direction with respect to X axis in the map for the wind.
 
-The default configuration file for the simulator is `CarlaWeather.ini` and comes 
-with the shipped simulator. This file loads up several pre-determined weather 
-profiles for editing in the Scenario Editor. These profiles can be edited in 
-the Scenario Editor and saved out to a `.json` format that is usable in the
-monoDrive clients.
+<p>&nbsp;</p>
 
-After pressing the "Save File Button" pictured above, two files can be found
-in the `Config` directory with the names specified in the 
-"Configuration File Name" field. In the `.ini` file, the saved profile can 
-be found under the specified "Profile Name" and appear as below:
+## Exporting Weather Descriptions
 
-In the `.json` file, the weather profile can be found in the JSON group where 
-the `id` field matches the "Profile Name":
+After exporting the weather descriptions you will receive a json file with name parity to the description you saw in the editor. It is structured as `descriptions` which points to a list of descriptions. `id` refers to the name which will be visible in the Weather Description drop down.
```

Comment:
Removing you : After exporting the weather descriptions, there will be a json file with name parity to the description you saw in the editor. It is structured as `descriptions` which points to a list of descriptions and `id` refers to the name which will be visible in the Weather Description drop down.

Reply:
fixed, i think